### PR TITLE
Finnish language support and new features

### PR DESCRIPTION
**Pull request by lumi-89**
Added:

- Finnish language locales
- setting for the format of location in the user profile and card
- option to add a show-map button in user profile and card
- option to show user avatars in user markers
- option to hide marker labels
- option to choose the language of location geocoding (user defined or site default)

### CODE REVIEW

File: assets/javascripts/discourse/connectors/user-card-location-and-website/replace-location.hbs [hunk 1]
```diff
@@ -1,3 +1,14 @@
 {{#if showUserLocation}}
-  <span class='map-location'>{{d-icon "map-marker"}} <span>{{user.custom_fields.geo_location.address}}</span></span>
+  {{mount-widget widget="user-location" args=user}}
 {{/if}}
+{{#if user.website_name}}
```
Comment (angusmcleod) [inline assets/javascripts/discourse/connectors/user-card-location-and-website/replace-location.hbs]: @lumi-89 Generally, it's not very sustainable to replicate core markup in a plugin as it'll break every time the core logic changes. What's the argument for creating a new user website markup? Is it to make it easier to manage the location and website DOM elements? If so, I suggest it's best to do this via css and / or than re-creating the core markup, even if that makes it initially easier.

Comment (lumi-89) [reply assets/javascripts/discourse/connectors/user-card-location-and-website/replace-location.hbs]: @angusmcleod I agree. I was only following the approach used for hiding the default location element, but I should really have revised that part of the code. I have done that now.

**Comment by angusmcleod** [comment]
@lumi-89 Thanks. I appreaciate the extent to which you've followed the existing coding style in the plugin. It makes it easy to review this.

**Comment by angusmcleod** [comment]
@lumi-89 Just a heads up that I've needed to update the user-location widget to handle a few different style issues. You can still reproduce the style you had in this PR with a theme component.

**Pull request merged**